llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -1023,6 +1023,37 @@ static void parseCondBranch(MachineInstr &LastInst, MachineBasicBlock *&Target,
   Cond.push_back(LastInst.getOperand(1));
 }
 
+static unsigned getReverseOpcode(unsigned Opcode) {
```

Comment:
Please give this a slightly more descriptive name. Maybe `getInverseXqcicmOpcode`? "Inverse" is the right term in LLVM for how the conditions have been changed, and this is only handling Xqcicm.